File: Primer-Bimestre/Segundo Parcial/Tarea4_EstCondRepetitivas/Tarea4_python/SerieFibonacci.py
print("")
# Inicialización de variables
a = 0; b = 1; c = 0; i = 1; n = 0

# Ingresar los datos
print("***Serie de Fibonacci***")
n = int(input("Ingrese la cantidad de elementos para la serie: "))

# Proceso
while (i <= n):
    print(a)
    c = a + b
    a = b
    b = c
    i = i + 1
    

File: Primer-Bimestre/Segundo Parcial/Tarea4_EstCondRepetitivas/Tarea4_java_DoWhile/src/tarea4do_while/SerieFibonacci.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package tarea4do_while;
import java.util.Scanner;
/**
 *
 * @author DELL
 */
public class SerieFibonacci {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
       // declarar e inicializar las variables
       int a, b, c, i, n;
       a = 0; b = 1; c = 0; i = 1; n = 0;
       
       // Ingresar datos
        System.out.println("***Serie de Fibonacci***");
        System.out.print("Ingrese cantidad de elementos para la serie: ");
        n = leer.nextInt();
        
       // Proceso
       
        do {
           System.out.println(a);
           c = a + b;
           a = b;
           b = c;
           i = i + 1;
        }while (i <= n);
    }
}
